integration/setup/setup.go

```diff
@@ -389,5 +312,65 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
+	{
+		azureClusterConfig := &v1alpha1.AzureClusterConfig{
```

Comment:
We need `cluster-operator` to react to `AzureClusterConfig` so it creates the required certificates. Before this PR, the certificates were being installed using e2etemplates.